### PR TITLE
migrate code from googleapis/nodejs-cloudbuild

### PR DESCRIPTION
**Pull request by grayside**
- feat: initial release of cloud build library (#2)
- feat(docs): add listBuildTriggers sample to README (#5)
- feat: moves library to TypeScript code generation (#10)
- chore: release 1.0.0 (#3)
- chore: release 1.1.0 (#20)
- chore: update license headers
- chore: release 1.2.0 (#25)
- chore: release 1.3.0 (#33)
- chore(deps): update dependency c8 to v7 (#38)
- refactor: use explicit mocha imports (#40)
- chore(deps): update dependency mocha to v7 (#43)
- chore: release 1.3.1
- chore: release 1.3.2 (#49)
- chore: release 1.3.3 (#59)
- chore: release 1.4.0 (#63)
- chore: release 1.5.0 (#67)
- chore(deps): update dependency mocha to v8 (#122)
- chore: release 2.0.0 (#92)
- chore: release 2.0.1 (#126)
- chore: release 2.0.2 (#133)
- chore: release 2.0.3 (#143)
- chore: release 2.0.4 (#179)
- chore: release 2.0.5 (#182)
- chore: release 2.0.6 (#186)
- chore: release 2.1.0 (#205)
- chore: release 2.2.0 (#213)
- chore: release 2.2.1 (#220)
- chore: release 2.2.2 (#227)
- chore: release 2.2.3 (#232)
- chore: release 2.2.4 (#242)
- chore: release 2.2.5 (#247)
- chore: release 2.2.6 (#250)
- chore: release 2.2.7 (#252)
- chore: release 2.3.0 (#263)
- chore: release 2.3.1 (#266)
- chore: release 2.3.2 (#269)
- chore: release 2.4.0 (#270)
- chore: release 2.5.0 (#272)
- chore: release 2.6.0 (#275)
- docs(samples): add example tags to generated samples (#287)
- chore(quickstart): mention private pool caveats (#317)
- build!: update library to use Node 12 (#325)
- chore(main): release 3.0.0 (#326)
- chore(main): release 3.0.1 (#328)
- chore(main): release 3.1.0 (#334)
- chore(main): release 3.1.1 (#340)
- chore(main): release 3.1.2 (#345)

### CODE REVIEW

File: .github/workflows/workflows.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
   "appengine/typescript",
   "appengine/websockets",
   "appengine/twilio",
+  "cloudbuild",
```
Comment (NimJay) [inline .github/workflows/workflows.json:20]: **Nitpick (feel free to ignore):**
Should the folder name be `cloud-build` to be consistent with existing `cloud-*` folders in this repo?

**Review by kweinmeister — APPROVED** [review]

**Comment by NimJay** [comment]
I have overridden the CLA check. ✅

**Review by NimJay — APPROVED** [review]

**Review by ace-n — CHANGES_REQUESTED** [review]
Two questions:
1) Does `quickstart.js` belong with the library (in the `google-cloud-node` repo)?
2) Do we need to update `CODEOWNERS`?

File: .github/workflows/workflows.json (hunk 1 above)
Comment (grayside) [reply .github/workflows/workflows.json:20]: I'm aligning with the the product API name and region tag, and think it's a minor bug where other directories don't but not necessarily worth fixing.

**Comment by grayside** [comment]
Per discussion, the quickstart should be forked because it's used in the docs. It will also move to the library as a test case.

**Review by ace-n — APPROVED** [review]

**Comment by ace-n** [comment]
Admin-merging to override CLA bot 👍

**Pull request merged**